src/Microsoft.Dnx.Tooling/NuGet/ContentModel/Infrastructure/Parser.cs

```diff
@@ -110,7 +110,7 @@ internal override bool TryMatch(ContentItem item, IDictionary<string, ContentPro
                 ContentPropertyDefinition propertyDefinition;
                 if (!propertyDefinitions.TryGetValue(_token, out propertyDefinition))
                 {
-                    throw new Exception(string.Format("Unable to find property definition for {{{0}}}", _token));
+                    throw new ArgumentException($"Unable to find property definition for '{_token}'");
```

Comment:
Actually I think I should probably not use an argument exception here since _token is technically not passed in to that method but is instead a member variable, would InvalidOperationException be better?